views/bookings/new.erb

```diff
@@ -1,3 +1,8 @@
+<!DOCTYPE html>
+<head>
+  <link rel="stylesheet" href="/styles.css">
+<h1>Maker House a Home</h1>
+</head>
```

Comment:
I suggest this is pulled out into  a layout.erb, the header and css config can then be applied to all pages. essenitally DRY this code up